sickrage/show/ComingEpisodes.py

```diff
@@ -158,3 +158,21 @@ def get_coming_episodes(categories, sort, group, paused=sickbeard.COMING_EPS_DIS
             grouped_results[category].append(result)
 
         return grouped_results
+
+    @staticmethod
+    def is_episode_later(data):
+        return data['localtime'].toordinal() >= (date.today() + timedelta(days=7)).toordinal()
+
+    @staticmethod
+    def is_episode_missed(data):
+        air_date = data['localtime']
+        runtime = data['runtime']
+
+        if runtime:
+            air_date += timedelta(minutes=runtime)
+
+        return air_date.toordinal() < date.today().toordinal()
+
+    @staticmethod
+    def is_episode_today(data):
+        return data['localtime'].toordinal() == date.today().toordinal()
```

Comment:
This last function is the only one I see that looks wrong at first glance xD.

is_episode_today is checking if the time is at exactly a day it looks like, no hours, no seconds.

I think you want something like:

```python
def is_episode_today(data):
        return (data['localtime'] - date.today()).days == 0
```